src/ctia/auth/jwt.clj

```diff
@@ -117,13 +121,26 @@
    (:casebook (all-entities))
    (:access scope-repr)))
 
+(defn gen-assets-capabilities
+  "Generate capabilities for the root-scope 'asset-intel'."
+  [scope-repr]
+  (if (= "asset-intel" (first (:path scope-repr)))
```

Comment:
I *think* you could loose the `if` statement

Reply:
It is there for the same reason as in the above comment https://github.com/threatgrid/ctia/pull/1078#discussion_r581281725. Without it, it will make assets available when only one of the required scopes present.

Reply:
If our answers to comment https://github.com/threatgrid/ctia/pull/1078#discussion_r581281725 are clear enough, you can now loose the `if` statement :-)